.travis.yml

```diff
@@ -1,185 +1,21 @@
 # Declare python as our language. This way we get our chosen Python version,
 # and pip is available. Gcc and clang are available anyway.
```

Comment:
This comment is no longer relevant, and we don't need `language: python` either.